README.md

```diff
@@ -50,6 +50,12 @@ set the initial deployment of your cluster:
 ceph-salt config
 ```
 
+This opens a shell where you can manipulate ceph-salt's configuration. Each
+configuration option is present under a configuration group. You can navigate
+through the groups and options using the familiar `ls` and `cd` commands
+similar to a regular shell. In each path you can type `help` to see the
+available commands. Different options might have different commands available.
+
```

Comment:
The only thing that comes to mind is:
1. Do you want this as a step-by-step instruction list?
2. If you don't, perhaps put this in a mark-up "Tip"?

Reply:
Based on what we have downstream @LenzGr - https://susedoc.github.io/doc-ses/master/html/ses-deployment/deploy-cephadm.html#deploy-cephadm-configure-shell I'd recommend a tip :)